src/utils.js

```diff
@@ -0,0 +1,9 @@
+class customError extends Error {
```

Comment:
Custom error makes me think of something like untitled error. Maybe call it ServerError? (The class name should start with uppercase)

Reply:
Good thinking, and thank you for the reminder about class naming conventions.